lib/search-summary/index.js

```diff
@@ -25,22 +25,75 @@ export default class SearchSummary extends Component {
     this.toggle();
     this.moveScrollToTop(this.props.window);
   }
+  handleOnChangeInputButtonClick () {
+    const {
+      numberOfAdults,
+      numberOfChildren,
+      duration
+    } = this.props;
+    this.toggle();
+    this.moveScrollToTop(this.props.window);
+    this.props.setNumberOfAdultsTitle(numberOfAdults);
+    this.props.setNumberOfChildrenTitle(numberOfChildren);
+    this.props.setDurationTitle(duration);
+    this.props.startSearch();
+  }
   render () {
+    const {
+      numberOfChildren,
+      numberOfAdults,
+      setNumberOfChildren,
+      childAge1,
+      childAge2,
+      childAge3,
+      childAge4,
+      setChildAge,
+      setNumberOfAdults,
+      setDepartureAirport,
+      setDepartureDate,
+      setDuration,
+      numberOfAdultsTitle,
+      numberOfChildrenTitle,
+      durationTitle,
+      departureAirport,
+      duration
+    } = this.props;
     const stateStyle = this.state.active ? 'changeInputActive' : 'changeInputInactive';
     const blueContainerStyle = this.state.active ? 'blueContainerActive' : 'blueContainer';
     const header = this.state.active ? 'headerInactive' : 'headerActive';
     const adultOptions = [1, 2, 3, 4, 5, 6];
```

Comment:
You could move these out into a separate options file to make this component a bit more readable